fe/src/pages/Header.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+function Header() {
+  return (
+    <>
+      <nav className="navbar navbar-expand-lg navbar-light bg-light border mb-3">
```

Comment:
- use react's fragment (apply to all related)
```suggestion
    <Fragment>
      <nav className="navbar navbar-expand-lg navbar-light bg-light border mb-3">
```